test/common/testlib.py

```diff
@@ -695,11 +695,11 @@ def logout(self):
 
         self.machine.allow_restart_journal_messages()
 
-    def relogin(self, path=None, user=None, superuser=None, wait_remote_session_machine=None):
+    def relogin(self, path=None, user=None, password=None, superuser=None, wait_remote_session_machine=None):
```

Comment:
Usually new parameters are appended, but I guess this is fine.

Reply:
yeah, but I felt like user and password should be next to each other as they are closely related